builders/toolkit/ethereum-api/precompiles/index.md

```diff
@@ -1,8 +1,65 @@
 ---
 title: Solidity Precompiles
 description: These guides showcase interacting with precompiles, which expose Substrate features via the easy-to-access Ethereum API of Tanssi EVM ContainerChains.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
 ---
+
+# Precompiles
+
+Precompiles are specialized smart contracts embedded in the blockchain, designed to perform complex operations more efficiently than standard contracts. In Tanssi EVM ContainerChains, precompiles expose Substrate functions through the accessible Ethereum API. These guides demonstrate how to interact with precompiles, allowing you to take advantage of advanced features easily.
+
+### Native Token ERC-20 
+
+Easily interact with your appchain's native token through the ERC-20 standard. This precompile simplifies operations such as:
+
+- Transferring tokens securely and efficiently.
+- Querying account balances with Ethereum-compatible tools.
+- Enabling seamless integration with other ERC-20-based systems.
+
+<div class="grid cards" markdown>
+
+-   :material-clock-fast:{ .lg .middle } __Native Token ERC-20 Precompile__
+
+    ---
+    
+    Handle and integrate your appchain’s native token using the ERC-20 interface, ensuring compatibility with Ethereum tools and wallets.
+    
+    [Manage ERC-20 Tokens](external-assets-erc20.md)
+
+</div>
+
+## Advanced Operations and Functionality
```

Comment:
It seems strange to list Native Token ERC-20 as it's own section then throw the rest here. I think we need to consider why someone would want to use each of these precompiles and use that to drive how these should be formatted and divided into categories